src/index.html

```diff
@@ -1,17 +1,123 @@
-<!doctype html>
-<html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
-  </head>
-  <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
-  </body>
-</html>
+<!doctype html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport"
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>HTML Form-01</title>
+  <link rel="stylesheet" href="./style.css">
+</head>
+
+<body>
+  <form action="" method="POST" autocomplete="off" onsubmit="onsubmit()">
+    <h1>HTML Form</h1>
+    <script type="text/javascript" src="./main.js"></script>
+
+    <fieldset class="block-information">
+      <legend>Personal information</legend>
+      <label for="" class="form-field" id="surname">
+        Surname:
+        <input type="text" name="surname" required>
+      </label>
+
+      <label for="" class="form-field" id="name">
+        Name:
+        <input type="text" name="name" required>
+      </label>
+
+      <label for="" class="form-field" id="age">
+        How old are you?
+        <input type="number" name="age" value="12" min="12" max="100" required>

```

Comment:
```suggestion
        <input
          type="number"
          name="age"
          value="12"
          min="12"
          max="100"
          required
        >
```
Можно форматировать вот так, если много атрибутов